onboarding/source/controller.c

```diff
@@ -92,9 +110,11 @@ static void ledTimerCallback(TimerHandle_t xTimer) {
 }
 
 static void lightTimerCallback(TimerHandle_t xTimer) {
+    ASSERT(xTimer != NULL);
     /* USER CODE BEGIN */
     // Send light event to light service queue
-
+    light_event_t *event = MEASURE_LIGHT;
```

Comment:
You'll need to first create a variable of type `light_event_t` with a value of `MEASURE_LIGHT`. `sendToLightServiceQueue` expects a pointer to that variable.

`light_event_t event = MEASURE_LIGHT;` and then pass in `&event`